### DISCUSSION TITLE
Error communicating with OpenAI "Exception: ⚠️ _An error has occurred._ ⚠️

### DISCUSSION THREAD

**Discussion by takada47** [n3d1117/chatgpt-telegram-bot#260] [Q&A] [answered]
Hi there, how to solve this problem ?
"Exception: ⚠️ _An error has occurred._ ⚠️
Error communicating with OpenAI"
I changed the token, checked it, checked the Openal balance, but nothing. how to fix it?

**Comment by n3d1117** [comment]
Hi @takada47 make sure that you're not using a proxy and you don't have any outbound traffic restrictions (are you using pythonanywhere.com?)

**Reply by takada47** [reply]
I don't use VPN and proxy. pythonanywhere.com I use

**Reply by n3d1117** [reply, marked as answer]
pythonanywhere's free tier has restricted outbound Internet access. I recommend using another host

**Comment by kaapo413** [comment]
hey ther, i have the same problem, but I have to use VPN as telegram is blocked in my country. does anyone have any suggestions?
